src/pyBiodatafuse/utils.py

```diff
@@ -126,6 +126,12 @@ def combine_sources(bridgedb_df: pd.DataFrame, df_list: List[pd.DataFrame]) -> p
 
     m = m.loc[:, ~m.columns.duplicated()]  # remove duplicate columns
 
+    # Ensure "Uniprot-TrEMBL" column is present
+    uniprot_trembl_df = bridgedb_df[bridgedb_df["target.source"] == "Uniprot-TrEMBL"]
+    uniprot_trembl_df = uniprot_trembl_df.groupby("identifier")["target"].apply(list).reset_index()
+    uniprot_trembl_df.rename(columns={"target": "Uniprot-TrEMBL"}, inplace=True)
+
+    m = pd.merge(m, uniprot_trembl_df, on="identifier", how="left")
```

Comment:
I think this is incorrect as you always have the protein as input. We need to make it flexible to have metabolites as input too

Reply:
Good catch, indeed this would make a metabolite query fail. Two options: 
- Surround with an `if  bridgedb_df["target.source"] == "Uniprot-TrEMBL"` to not fail if there are no Uniprot-TrEMBL items coming from the bridgedb dataframe
- Delete completely and not keep variants in the KG

I assume we'll go for the second option?
@YojanaGadiya

Reply:
Yes. Lets go with the second option for now.. or another idea could be to have a param in the function for gene/metabolite and this code can go in for gene

Reply:
Solved, not using this anymore!